src/components/Dialog.tsx

```diff
@@ -67,37 +64,46 @@ export function Provider({ value, children }: ProviderProps) {
 const OnCloseContext = createContext<(() => void) | undefined>(undefined)
 
 const HeaderRow = styled(Row)`
+  display: flex;
   height: 1.75em;
-  margin: 0 0.75em 0.75em;
-  padding-top: 0.5em;
   ${largeIconCss}
 
+  justify-content: flex-start;
+  margin: 0.5em 0.75em 0.75em;
+  position: relative;
+
   button {
     height: ${({ iconSize }) => iconSize}em;
   }
 `
 
+const StyledBackButton = styled(ArrowLeft)`
+  :hover {
+    cursor: pointer;
+    opacity: 0.6;
+  }
+`
+
+const Title = styled.div`
+  left: 50%;
+  position: absolute;
+  top: 50%;
```

Comment:
i could see why we want absolute positioning on the horizontal axis, but should we stick with flex / `align-items: center` for vertical spacing here?

Reply:
nit / non blocking

Reply:
excellent point, will do ty